resonate/options.py

```diff
@@ -56,6 +58,7 @@ def merge(
             durable=durable if durable is not None else self.durable,
             id=id if id is not None else self.id,
             idempotency_key=idempotency_key if idempotency_key is not None else self.idempotency_key,
+            non_retriable_errors=non_retriable_errors if non_retriable_errors is not None else self.non_retriable_errors,
```

Comment:
Mind if we go with `non_retryable_exceptions`?

It looks like "retryable" and "retriable" are both correct spelling but retryable seems more widely used, and in python errors are called exceptions